### PR TITLE
Major upgrade merge to main

### PR DESCRIPTION
**Pull request by halsaphi**
PR to publish updated docs - from the major-upgrade branch to main.

### CODE REVIEW

File: docs/about-mina/faq.mdx [hunk 1]
```diff
@@ -50,8 +46,11 @@ Mina's consensus mechanism is an implementation of Ouroboros proof of stake. Due
 
 Yes, check out these block explorers:
 
-- https://minascan.io
-- https://minataur.net 
+- [Mina Explorer](https://minaexplorer.com/)
```
Comment (shimkiv) [inline docs/about-mina/faq.mdx:49]: Let's remove Mina Explorer if it won't support mainnet after the HF.

File: docs/participate/careers.mdx [hunk 2]
```diff
@@ -8,12 +8,12 @@ If you're looking for another way to get involved with Mina, consider exploring
...
 ### Mina Ecosystem Partners
 
-- O(1) Labs [Careers](https://boards.greenhouse.io/o1labs) - Incubators of the Mina Protocol
+- o1Labs [Careers](https://boards.greenhouse.io/o1labs) - Incubators of the Mina Protocol
 - =nil; Foundation [Careers](https://nil.foundation/careers/) - Creators of the zkBridge bridge from Mina to Ethereum
 - Viable Systems [Careers](https://www.viablesystems.io/rust-jobs) - Tools to help improve the performance of the Mina nodes and overall network
 
-Are you a Mina ecosystem partner? Select __EDIT THIS PAGE__ at the top of this page to submit a pull request to add your company blog. Thank you for helping grow the Mina ecosystem.
\ No newline at end of file
+Are you a Mina ecosystem partner? Select __EDIT THIS PAGE__ at the top of this page to submit a pull request to add your company blog. Thank you for helping grow the Mina ecosystem.
```
Comment (shimkiv) [inline docs/participate/careers.mdx:19]: `__EDIT THIS PAGE__`

File: docs/using-mina/how-to-use-zkapp.mdx [hunk 3]
```diff
@@ -5,16 +5,15 @@ hide_title: true
 
 import Subhead from '@site/src/components/common/Subhead';
 
-:::info
```
Comment (shimkiv) [inline docs/using-mina/how-to-use-zkapp.mdx:8]: Restore the info tip please.

File: docs/zkapps/writing-a-zkapp/feature-overview/custom-tokens.mdx [hunk 4]
```diff
@@ -23,6 +23,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/writing-a-zkapp/feature-overview/custom-tokens.mdx:26]: No need in this extra space.

File: docs/zkapps/tutorials/10-account-updates.mdx [hunk 5]
```diff
@@ -23,6 +23,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/tutorials/10-account-updates.mdx:26]: No need in this extra space.

File: docs/zkapps/tutorials/07-oracle.mdx [hunk 6]
```diff
@@ -22,6 +22,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/tutorials/07-oracle.mdx:25]: No need in this extra space.

File: docs/zkapps/tutorials/03-deploying-to-a-network.mdx [hunk 7]
```diff
@@ -24,6 +24,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/tutorials/03-deploying-to-a-network.mdx:27]: No need in this extra space.

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx [hunk 8]
```diff
@@ -24,6 +24,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx:27]: No need in this extra space.

File: docs/zkapps/o1js/recursion.mdx [hunk 9]
```diff
@@ -20,6 +20,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/o1js/recursion.mdx:23]: No need in this extra space.

File: docs/zkapps/o1js/merkle-tree.mdx [hunk 10]
```diff
@@ -17,6 +17,7 @@ zkApp programmability is not yet available on the Mina **Mainnet**, but zkApps c
 
 :::
 
+
```
Comment (shimkiv) [inline docs/zkapps/o1js/merkle-tree.mdx:20]: No need in this extra space.

File: docs/zkapps/how-to-deploy-a-zkapp.mdx [hunk 11]
```diff
@@ -0,0 +1,114 @@
+---
```
Comment (shimkiv) [inline docs/zkapps/how-to-deploy-a-zkapp.mdx:1]: Make sure it 100% corresponds to: https://docs.minaprotocol.com/zkapps/writing-a-zkapp/introduction-to-zkapps/how-to-deploy-a-zkapp
(including content itself + correct belonging to the docs structure)

File: docs/zkapps/o1js-reference/README.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
+
```
Comment (shimkiv) [inline docs/zkapps/o1js-reference/README.md:1]: Make sure it 100% corresponds to what we currently have for https://docs.minaprotocol.com/
(including content itself + correct belonging to the docs structure)
Readme is outdated and most likely it no longer belongs to this docs path.

File: docs/zkapps/o1js/custom-tokens.mdx [hunk 13]
```diff
@@ -0,0 +1,211 @@
+---
```
Comment (shimkiv) [inline docs/zkapps/o1js/custom-tokens.mdx:1]: Make sure it 100% corresponds to what we currently have for https://docs.minaprotocol.com/
(including content itself + correct belonging to the docs structure)
Custom tokens is no published for the users I think.

File: docs/zkapps/testing-zkapps-lightnet.mdx [hunk 14]
```diff
@@ -0,0 +1,346 @@
+---
```
Comment (shimkiv) [inline docs/zkapps/testing-zkapps-lightnet.mdx:1]: Make sure it 100% corresponds to what we currently have for https://docs.minaprotocol.com/zkapps/writing-a-zkapp/introduction-to-zkapps/testing-zkapps-lightnet
(including content itself + correct belonging to the docs structure)
Document is outdated and most likely it no longer belongs to this docs path.

**Review by shimkiv — CHANGES_REQUESTED** [review]

File: docs/about-mina/faq.mdx (hunk 1 above)
Comment (halsaphi) [reply docs/about-mina/faq.mdx]: removed minaexplorer

File: docs/zkapps/how-to-deploy-a-zkapp.mdx (hunk 11 above)
Comment (halsaphi) [reply docs/zkapps/how-to-deploy-a-zkapp.mdx:1]: was duplicated - I've removed the extra one

File: docs/zkapps/o1js/custom-tokens.mdx (hunk 13 above)
Comment (halsaphi) [reply docs/zkapps/o1js/custom-tokens.mdx:1]: Not displayed - see sidebar.js

File: docs/zkapps/testing-zkapps-lightnet.mdx (hunk 14 above)
Comment (halsaphi) [reply docs/zkapps/testing-zkapps-lightnet.mdx:1]: again a duplicate not displayed - as per sidebar.js - I've removed it.

File: docs/zkapps/o1js-reference/README.md (hunk 12 above)
Comment (halsaphi) [reply docs/zkapps/o1js-reference/README.md:1]: again a duplicate - I've removed.

File: docs/participate/careers.mdx (hunk 2 above)
Comment (halsaphi) [reply docs/participate/careers.mdx]: changed __ to "

File: docs/using-mina/how-to-use-zkapp.mdx (hunk 3 above)
Comment (halsaphi) [reply docs/using-mina/how-to-use-zkapp.mdx:8]: restored

File: docs/zkapps/writing-a-zkapp/feature-overview/custom-tokens.mdx (hunk 4 above)
Comment (halsaphi) [reply docs/zkapps/writing-a-zkapp/feature-overview/custom-tokens.mdx]: removed

File: docs/zkapps/tutorials/10-account-updates.mdx (hunk 5 above)
Comment (halsaphi) [reply docs/zkapps/tutorials/10-account-updates.mdx]: removed

File: docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx (hunk 8 above)
Comment (halsaphi) [reply docs/zkapps/tutorials/02-private-inputs-hash-functions.mdx]: removed

File: docs/zkapps/o1js/merkle-tree.mdx (hunk 10 above)
Comment (halsaphi) [reply docs/zkapps/o1js/merkle-tree.mdx]: removed

**Review by shimkiv — APPROVED** [review]

**Pull request merged**